src/kirby/components/badge/badge.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit, Input } from '@angular/core';
+
+@Component({
+  selector: 'kirby-badge',
+  templateUrl: './badge.component.html',
+  styleUrls: ['./badge.component.scss'],
+})
+export class BadgeComponent implements OnInit {
+  @Input() colortype: string = 'danger';
```

Comment:
Consider limitting to possible colortypes (see: https://github.com/kirbydesign/designsystem/blob/master/src/kirby/components/icon/icon.component.ts#L30)

Reply:
Added